### PR TITLE
Adding tables

### PR DESCRIPTION
**Pull request by mhines-usgs**
- [x]  agency
- [x] aquifer
- [x] ?? national aquifer ?? 
- [x] country
- [x] state
- [x] county
- [x] minor civil division
- [x] hydrologic unit
- [x] horizontalAccuracyCode
- [x] horizontalDatum
- [x] horizontalMethodCode
- [ ] altitudeAccuracy (no table I believe, this is an integer value in the monitoring_location table)
- [x] altitudeDatum
- [x] altitudeMethodCode
- [x] use
- [x] monitoring location type

### CODE REVIEW

File: mlr-database-schema/src/main/liquibase/aquifer.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.0.xsd" logicalFilePath="liquibaseChangeSet/aquifer.xml">
+    <changeSet author="mhines" id="add_aquifer_table">
```
Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: unless @dpattermann-usgs or @kmschoep-usgs has more clarity on how the two aquifer tables differ, I followed Dominick's lead in the model file..)

Comment (dpattermann-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: I'm not really sure if there will be more or not. We might need the national aquifer in the database alongside the other instances of different aquifers? I'm not entirely sure.

Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: ok, i think we can presume one way and then follow up with Jim Kreft or someone who is more familiar with the data for review?

Comment (dpattermann-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: Agreed. I say we pick a direction and then fix it if we need to change it (unless someone else feels differently). 👍

Comment (kmschoep-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: Based on the data in natdb, the national aquifer and aquifer tables are entirely separate with different data in them, and the existing sitefile table has both aquifer and national aquifer columns.   I think we need a model for both aquifer and national aquifer unless aquifer can be overloaded and handle both.

File: mlr-database-schema/src/main/liquibase/aquifer.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+		<createTable tableName="aquifer">
+			<column name="id" type="INT4">
+				<constraints nullable="false" unique="true"/>
+			</column>
+			<column name="name" type="text" />
+			<column name="code" type="text" />
+			<column name="nationalaquifer" type="text" />
+		</createTable>
+		<addForeignKeyConstraint baseColumnNames="id" baseTableName="aquifer" constraintName="fk_aquifer_id" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="aquifer_id" referencedTableName="monitoring_location"/>
+		<addForeignKeyConstraint baseColumnNames="nationalaquifer" baseTableName="national_aquifer" constraintName="fk_national_aquifer_id" deferrable="false" initiallyDeferred="false" onDelete="CASCADE" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="national_aquifer"/>
```
Comment (mhines-usgs) [inline mlr-database-schema/src/main/liquibase/aquifer.xml]: not sure about this and the national_aquifer table, please check it out @kmschoep-usgs

**Pull request merged**